core/App/ehsm_napi.h

```diff
@@ -322,6 +292,12 @@ char* NAPI_Verify(const char* cmk_base64,
         const char* signature_base64);
 
 
+char* NAPI_RA_HANDSHAKE_MSG0(const char* request);
```

Comment:
what's the request structure? please add comments to decript the details about the structure.

Reply:
It will be fix